README.md

```diff
@@ -59,12 +59,13 @@ To prepare your system to deploy Clearwater manually, run:
 
 To start the Clearwater services, run:
 
-    sudo docker run -d --name homestead -p 22 clearwater/homestead
-    sudo docker run -d --name homer -p 22 clearwater/homer
-    sudo docker run -d --name ralf -p 22 clearwater/ralf
-    sudo docker run -d --name sprout -p 22 --link homestead:homestead --link homer:homer --link ralf:ralf clearwater/sprout
-    sudo docker run -d --name bono -p 22 -p 3478:3478 -p 3478:3478/udp -p 5060:5060 -p 5060:5060/udp -p 5062:5062 --link sprout:sprout clearwater/bono
-    sudo docker run -d --name ellis -p 22 -p 80:80 --link homestead:homestead --link homer:homer clearwater/ellis
+    sudo docker network create --driver bridge clearwater_nw
```

Comment:
How does this work under Docker-based platforms (e.g. Kubernetes)?  I believe the old model worked fine with Kubernetes.  Does the new one?